README.md

```diff
@@ -18,6 +18,8 @@ pip install https://github.com/open-craft/xblock-html
 ```
 You may specify the `-e` flag if you intend to develop on the repo.
 
+Note that as of version 0.2.0, Python 2.7 is no longer supported. The current minimum Python version is 3.5.
+
```

Comment:
@taniwha, the `README` file doesn't give the full installation/usage instructions - for example, it doesn't specify what needs to be added to the Studio advanced modules list to be able to use the HTML5 and the excluded HTML5 XBlocks. Can you add the details for those?